e2e/cypress/integration/messaging/message_draft_then_switch_channel_spec.js

```diff
@@ -24,46 +22,57 @@ describe('Message Draft and Switch Channels', () => {
     });
 
     it('MM-T131 Message Draft Pencil Icon - CTRL/CMD+K & "Jump to"', () => {
-        // # In a test channel, type some text in the message input box
-        // # Do not send the post
-        cy.get('#sidebarItem_town-square').click({force: true});
+        const testChannel = 'Off-Topic';
+        const message = 'message draft test';
 
-        // * Validate if the channel has been opened
-        cy.url().should('include', `/${testTeam.name}/channels/town-square`);
+        // * Validate if the draft icon is not visible at LHS before making a draft
+        verifyDraftIcon(testChannel, false);
 
-        // * Validate if the draft icon is not visible on the sidebar before making a draft
-        cy.get('#publicChannel').scrollIntoView();
-        cy.get('#sidebarItem_town-square #draftIcon').should('be.not.visible');
+        // # Go to test channel and check if it opened correctly
+        openChannelFromLhs(testTeam.name, testChannel);
 
-        // Type in some text into the text area of the opened channel
-        cy.get('#post_textbox').type('message draft test');
+        // # Type a message in the input box but do not send
+        cy.findByLabelText(`write to ${testChannel.toLowerCase()}`).should('be.visible').type(message);
 
-        // # Switch to another channel
-        cy.get('#sidebarItem_off-topic').click({force: true});
+        // # Switch to another channel and check if it opened correctly
+        openChannelFromLhs(testTeam.name, 'Town Square', 'town-square');
 
-        // * Validate if the newly navigated channel is open
-        cy.url().should('include', `/${testTeam.name}/channels/off-topic`);
+        // * Validate if the draft icon is visible at LHS
+        verifyDraftIcon(testChannel, true);
 
-        // # Press CTRL/CMD+K
+        // # Press CTRL/CMD+K shortcut to open Quick Channel Switch modal
         cy.typeCmdOrCtrl().type('K', {release: true});
 
-        // * Click on hint in modal to get out of overlapping suggestion list
-        cy.get('#quickSwitchHint').click();
+        // * Verify that the switch model is shown
+        cy.findAllByRole('dialog').first().findByText('Switch Channels').should('be.visible');
 
         // # Type the first few letters of the channel name you typed the message draft in
-        cy.get('#quickSwitchInput').type('tow');
-        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('#quickSwitchInput').type(testChannel.substring(0, 3));
```

Comment:
```suggestion
        cy.findByRole('textbox').type(testChannel.substring(0, 3));
```

Reply:
@saturninoabril 
All look good to me, except one suggestion :)

Reply:
Thanks for the review. Yeah, I should've done that :). I've added an aria-label and so this could be `cy.findByRole('textbox', {name: 'quick switch input'})`.  Please take a look again.